docs/source/developers/python.rst

```diff
@@ -563,19 +563,25 @@ Then run the unit tests with:
    the Python extension. This is recommended for development as it allows the
    C++ libraries to be re-built separately.
 
-   If you want to bundle the Arrow C++ libraries with ``pyarrow``, add
-   the ``--bundle-arrow-cpp`` option when building:
+   If you are using conda package manager then conda will ensure Arrow C++
+   libraries are found. In case you are *not* using conda then you have to
+
+   * add the path of installed DLL libraries to ``PATH`` every time before
+     importing ``pyarrow``, or
+   * bundle the Arrow C++ libraries with ``pyarrow``.
+
+   If you want to bundle the Arrow C++ libraries with ``pyarrow``, use
+   ``PYARROW_BUNDLE_ARROW_CPP`` environment variable before building pyarrow:
 
    .. code-block::
 
-      $ python setup.py build_ext --bundle-arrow-cpp
+      $ pushd arrow\python
+      $ export PYARROW_BUNDLE_ARROW_CPP=1
+      $ python -m pytest pyarrow
```

Comment:
This should be building pyarrow? (not testing) So `python setup.py build_ext --inplace`? (that is what is being used above) or a pip install

Reply:
Yea, of course. Sloppy again :( will correct.